src/pacstall.py

```diff
@@ -41,4 +65,59 @@
 ]:
     message.fancy("info", "APT lists were updated more than a week ago")
     message.fancy("info", "Updating APT lists")
-    os.system("sudo apt-get -qq update")
+    os.system("/usr/bin/sudo /usr/bin/apt-get -qq update")
```

Comment:
Why hardcoded paths? Does it not work without?

Reply:
How can you be sure that `/usr/bin/sudó` exists?

Reply:
Because it does? Duh.